src/components/Scorecards/ScorecardsServicePage/ScorecardsServiceProgress.tsx

```diff
@@ -113,7 +113,7 @@ export const ScorecardsServiceProgress = ({
           <Button
             variant="contained"
             color="primary"
-            href={cortexScorecardPageURL(scorecardId)}
+            href={cortexScorecardPageURL({scorecardId})}
```

Comment:
Is prettier not working? 
```suggestion
            href={cortexScorecardPageURL({ scorecardId })}
```

Reply:
There is no prettier in this repository.

Reply:
😢

Reply:
I ran it manually and checked in the changes.